chsdi/models/vector/bafu.py

```diff
@@ -11,6 +11,22 @@
 Base = bases['bafu']
 
 
+class Hydrogeologischekarte100(Base, Vector):
+    __tablename__ = 'hydrogeologische_karte_100'
+    __table_args__ = ({'schema': 'hydrologie', 'autoload': False})
+    __bodId__ = 'ch.bafu.hydrogeologische-karte_100'
+    __queryable_attributes__ = ['name', 'pdf_list']
+    __template__ = 'templates/htmlpopup/hydrogeologische-karte_100.mako'
+    __label__ = 'name'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    name = Column('raster_name', Text)
+    pdf_list = Column('pdf_list', Text)
+    the_geom = Column(Geometry(geometry_type='GEOMETRY',
```

Comment:
ouch, I missed this new class. Very nice!
It would make sense to send the integration team a heads-up for those things. It's difficult to see every change in our code